io.github.retgal.Dayon.yml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.retgal.Dayon
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: dayon_assisted.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --filesystem=home
+  - --filesystem=/tmp
```

Comment:
What is this one needed for?

Reply:
The temp directory is used during the file transfer.

Reply:
The app has access to sandboxed /tmp by default. If it's only about using it as temporary location, it can be dropped.

Reply:
Sorry but no, I tested it and file transfer failed without having access to /tmp.

Reply:
@barthalion  Sorry but no, I tested it and file transfer fails without having access to the "real" /tmp. References to the transfered files are copied to the recipients clipboard. Later on, when the user is pasting the clipboard's content, the files have to be accessed again - and that's when the error occurs. (file not found)